webapi/src/main/scala/org/knora/webapi/store/triplestore/impl/TriplestoreServiceLive.scala

```diff
@@ -368,11 +368,15 @@ case class TriplestoreServiceLive(
     val sparqlQuery = (graph: String) => s"DROP GRAPH <$graph>"
 
     for {
-      _      <- ZIO.logInfo("==>> Drop All Data Start")
+      _      <- ZIO.logWarning("==>> Drop All Data Start")
```

Comment:
to make it more visible on data dog? what about of end of dropping?

Reply:
Yes, wanted to make it visible but didn't want to spam the warnings channel